addons/website/static/src/js/content/menu.js

```diff
@@ -526,7 +526,10 @@ publicWidget.registry.menuDirection = publicWidget.Widget.extend({
      * @override
      */
     start: function () {
-        this.defaultAlignment = this.$el.is('.ml-auto, .ml-auto ~ *') ? 'right' : 'left';
+        this.rtl = !!this.el.closest('#wrapwrap.o_rtl');
```

Comment:
Since it is a boolean, maybe name it `isRTL` or something like that so it is more clear :slightly_smiling_face:  
Also, why do you put it in `this` instead of just declaring a `const` (since it is only used here) ?

Reply:
Good catch :+1: